challenges/objects-arrays.js

```diff
@@ -89,31 +129,55 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+const displayScientific = zooAnimals.forEach(info => {(animalNames.push(`${info.scientific_name} ${info.animal_name}`)); 
+});
+
 console.log(animalNames);
 
 /* Request 2: .map()    
 
-The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
+The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  
+Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
 const lowerCase = [];
+const animalNameLower = zooAnimals.map((info) => {
+  lowerCase.push(`${info.animal_name}`.toLowerCase());
+})
+
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
-The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
+The zoos are concenred about animals with a lower population count. 
+Find out which animals have a population less than 5.
 
 */
 const lowerPopulation = [];
+
+const lowerPop = zooAnimals.filter(info => {
+  lowerPopulation.push(info.animal_name, info.population < 5);
+});
+
```

Comment:
Again, some work is still missing here. Look up the `filter()` syntax and try to understand how it works. `push` is not needed in this case.

[Here's a short read on filter method with a really simple example](https://javascript.info/array-methods#filter). See how you could improve this one.